src/ripple/protocol/BuildInfo.h

```diff
@@ -68,6 +68,18 @@ getFullVersionString();
 std::uint64_t
 getEncodedVersion();
 
+/** Compare the local node's rippled server software version with the given one.
+
+    @note We only compare the major version, minor version, and patch version.
+    @param othersVersion another node's rippled server software version
+    @return true if local node's rippled server software version is lower,
+            false otherwise.
+*/
+bool
+localVersionLow(std::uint64_t othersVersion);
```

Comment:
This name is slightly confusing, at least to me. I'd suggest renaming this to `isNewerVersion` instead.

Also, the comment needs to explain that this function only understands version numbers that are generated by `rippled`. It's true that there are no other implementations currently but ~if~ when they become available, they'll likely have a different versioning scheme, and we couldn't directly compare against them.

Reply:
Right, renamed the function and added a short note about "this function only understands version numbers that are generated by rippled."